doc/bash/symfony_aliases.md

```diff
@@ -40,4 +40,20 @@
 - **sfai**: `php app/console assets:install web --symlink`
 
 ### Symfony server ###
-- **sfs**: `php app/console se:ru`
+- **sfsr**: `php app/console server:run`
+- **sfss**: `php app/console server:status`
+- **sfsq**: `php app/console server:stop`
+
+### Doctrine ###
+docdbcreate=`doctrine:database:create`
```

Comment:
Please prefix all the lines from this group with `- ` (dash followed by space), like in the above groups.